packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -395,6 +397,45 @@ class ConfigCommand extends FlutterFireCommand {
       );
     }
 
+    if (iosOptions != null) {
+      final googleServiceInfoFile = path.join(
+        flutterApp!.iosDirectory.path,
+        'Runner',
+        iosOptions.optionsSourceFileName,
+      );
+
+      final file = File(googleServiceInfoFile);
+
+      if (!file.existsSync()) {
+        await file.writeAsString(iosOptions.optionsSourceContent);
+      }
+
+      final pathToScript = path.split(Platform.script.toFilePath());
+
+      final sourceDirIndex = pathToScript.indexOf('.dart_tool');
+
+      final listToPbxScriptDir = pathToScript.sublist(0, sourceDirIndex);
+
+      final pathToPbxScript = path.joinAll(
```

Comment:
Could you explain what these are doing sorry? Why is there a `.dart_tool` directory in Platform.script? shouldn't it be the full path to the FF CLI in .pub-cache dir being returned there?

Reply:
@Salakar 

`pathToScript` - logs out 
```bash
[/, Users, russellwheatley, projects, flutterfire_cli, .dart_tool, pub, bin, flutterfire_cli_monorepo, flutterfire_dev.dart-2.17.5.snapshot]
````
 `.dart_tool` is used to get to the root of the project so we can get the path to the ruby script (`listToPbxScriptDir`). Happy to change if you prefer another method.

I've just had a look in the `.pub-cache`, the ruby script is not there. But maybe I'm misunderstanding.

Reply:
Probably could move the script to inside the flutterfire_cli src package though?